src/node.js

```diff
@@ -19,7 +19,15 @@ const GRACE_PERIOD = 7500 // amount of ms to wait before sigkill
 function findIpfsExecutable () {
   const rootPath = process.env.testpath ? process.env.testpath : __dirname
 
-  const appRoot = path.join(rootPath, '..')
+  let appRoot = path.join(rootPath, '..')
+  // If inside <appname>.asar try to load from .asar.unpacked
+  // this only works if asar was built with
+  // asar --unpack-dir=node_modules/go-ipfs-dep/* (not tested)
+  // or
+  // electron-packager ./ --asar.unpackDir=node_modules/go-ipfs-dep
+  if (appRoot.includes(`.asar${path.sep}`)) {
+    appRoot = appRoot.replace(`.asar${path.sep}`, `.asar.unpacked${path.sep}`)
+  }
```

Comment:
Could you add this to the README too? It is fine to leave it here, but I want to make sure it this note is searchable and linkable to

Reply:
Sure